google/ggrc-core
# GGRC-8791 Primary Contacts and Secondary Contacts are not hidden from GGRC UI for Scope objects

# Issue description

Primary Contacts and Secondary Contacts are not hidden from GGRC UI for Scope objects. The reason is that Primary Contacts and Secondary Contacts were never removed from the DB. 

# Steps to test the changes

1. Run the migrations;
2. Login into GGRC app;
3. Open any Scope object;
4. Make sure there is no 'Primary Contacts' and 'Secondary Contacts' attributes.

# Solution description

The solution includes a migration which removes 'Primary Contacts' and 'Secondary Contacts' ACRs associated with the Scope objects. Related ACLs and ACPs are removed as well.

The migration also adds records for `objects_without_revisions` for all removed objects, as well for all scope objects.

# Sanity checklist


# Migration checklist

# PR Review checklist
